gradle.properties

```diff
@@ -16,7 +16,7 @@
 group=org.apache.kafka
 # NOTE: When you change this version number, you should also make sure to update
 # the version numbers in tests/kafkatest/__init__.py and kafka-merge-pr.py.
-version=0.11.0.0-SNAPSHOT
```

Comment:
Per the comment above, shouldn't kafka-merge-pr.py also be changed to update `DEFAULT_FIX_VERSION`? Also, not sure if we want to include slightly more code changes into these bumps, but I think `tests/kafkatest/version.py` will need updates at some point too.